### DISCUSSION TITLE
Pricing for mapped tasks

### DISCUSSION THREAD

**Discussion by th0ger** [PrefectHQ/prefect#5296] [Q&A]
The prefect success-based princing reads:

> Pay for what you use.
> Pricing is based on successful task runs per month.
> Never pay for retries, failures, or tasks that run in less than one second.

It is not clear whether mapped tasks are billled as a single parent task, or for each child task?

**Comment by zanieb** [comment]
Hey @th0ger — each mapped child is its own task run. Feel free to reach out to sales@prefect.io for more information.
